src/NServiceBus.NHibernate/Outbox/OutboxCleaner.cs

```diff
@@ -21,7 +23,7 @@ protected override Task OnStart(IMessageSession busSession)
 
             if (configValue == null)
             {
-                timeToKeepDeduplicationData = TimeSpan.FromDays(7);
+                timeToKeepDeduplicationData = settings.GetOrDefault<TimeSpan?>("Outbox.TimeToKeepDeduplicationData") ?? TimeSpan.FromDays(7);
```

Comment:
I think in most similar classes we tend to put the setting access code in the feature and pass the actual value to the startup task. It would keep the FST cleaner.